### DISCUSSION TITLE
reComputerJ1020v2, J20 Series| J202 | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#790] [Q&A]
# reComputerJ1020v2, J20 Series| J202 | Seeed Studio Wiki

reComputerJ2021 | J202

https://wiki.seeedstudio.com/reComputer_J2021_J202_Flash_Jetpack/

**Comment by SwiftGust** [comment]
Please add updated instruction for how to flash Jetpack 5.1 and later on Jetson Xavier NX (reComputer J20 series)
